keep-ui/app/(keep)/deduplication/DeduplicationTable.tsx

```diff
@@ -120,7 +120,9 @@ export const DeduplicationTable: React.FC<DeduplicationTableProps> = ({
       "Represents the percentage of alerts successfully deduplicated. Higher values indicate better deduplication efficiency, meaning fewer redundant alerts.",
   };
 
-  function resolveDeleteButtonTooltip(deduplicationRule: DeduplicationRule): string {
```

Comment:
This file is just re-formatted, right?

Reply:
Yes, this is outcome of running prettier